src/Generating/BioGen.h

```diff
@@ -48,31 +48,37 @@ class cBioGenCache :
 	typedef cBiomeGen super;
 
 public:
+
 	cBioGenCache(cBiomeGenPtr a_BioGenToCache, size_t a_CacheSize);
-	virtual ~cBioGenCache() override;
+
```

Comment:
Perhaps keep the destructor as `override` and `= default` to ensure the base class has a `virtual` destructor?
```suggestion
	virtual ~cBioGenCache() override = default;
```

Reply:
Does gcc 4.8 support that? (It's still used for RasPi cross-builds on the build server)

Reply:
According to [cppreference](https://en.cppreference.com/w/cpp/compiler_support) it's supported defaulted and deleted functions since gcc 4.4

Reply:
Changed.